src/jira/client/jira-client-issue-key-helper.ts

```diff
@@ -0,0 +1,172 @@
+import { uniq } from "lodash";
```

Comment:
Most of these are just moved from jiraclient into this file. I will highlight any other changes